### ISSUE TITLE
Kale's vitamin C content is 120mg per 100g

### ISSUE THREAD

**Issue by diraneyya** [WildCodeSchool/2023-01-EN-Berlin-Remote2-Markdown#1]
https://github.com/WildCodeSchool/2023-01-EN-Berlin-Remote2-Markdown/blob/34abc0c0fab5a19e3c67b51639f4a2f70d5e9ee1/yevhenii-airapetian.md?plain=1#L34

Please check sources[^1][^2]

[^1]: USDA https://fdc.nal.usda.gov/fdc-app.html#/food-details/168421/nutrients
[^2]: https://www.webmd.com/food-recipes/kale-nutrition-and-cooking

**Comment by DavidJFigueroa** [comment]
Issue fixed

**Comment by diraneyya** [comment]
Thanks for rescuing Kale. 🥬

**Comment by yevheniiairapetian** [comment]
Sorry, I didn't get what you mean.

ср, 18 Янв 2023 г., 19:01 Orwa Diraneyya ***@***.***>:

**Comment by diraneyya** [comment]
Hello @padfoo I was innocently (and playfully) trying to show you that we can have comments and replies in issues threads on GitHub as most people do not know about this feature when they start a technology bootcamp.

Obviously the amount of Vitamin C in Kale is not important. It was just a way for me to teach you and your classmates how to:
1. Use modern AI technology to fact check things (in this case the GPT-3 model from OpenAI)
2. How to create an issue from a single line, in which case the Kale Vitamin C content was a perfect match for this teaching opportunity
3. How to make a required change, create a pull request and merge
4. How to talk about the change in the issues thread, thanking people for their contribution and adding additional reflections and thoughts

Please do not take things personally. We are still trying to get to know each other and I am still trying to learn more about you.

If you feel bad about something, please share openly and transparently in class, this is the only way we can all feel safe, cherished, and motivated to learn together 🥰🥰🥰

**Comment by yevheniiairapetian** [comment]
Hello Orwa and Olga,
First of all, nobody has written to the Jobcenter. I came to these courses
to study and not receive jokes which I don't understand. I have written to
Orwa in the chat, he was online, and he didn't reply. On the next day I saw
the laughing-to-tears smile to my message. I don't think this is adult
behavoir either. I was stressed and could not speak with a cemera on lesson
and with you and Olga. I have also written to Olga about the camera issue
that I have (I feel stressed when it's not and can't concentrate) but I
did't receive any response for a week or so. My advisor knows the situation
very well because he helps me with things while I'm here in Germany. I see
poor communication, so I asked him to write on my behalf. But again, nobody
has written to Jobcenter. It is not smart to expect the same attitude
towards joking in class from every person.

чт, 19 Янв 2023 г., 12:06 Orwa Diraneyya ***@***.***>:
